core-tests/src/test/java/org/mule/runtime/core/internal/util/message/ResultToMessageIteratorTestCase.java

```diff
@@ -60,7 +60,7 @@ private ResultToMessageIterator createResultToMessageIterator() {
     CursorProviderFactory cursorProviderFactory = mock(CursorProviderFactory.class);
     BaseEventContext eventCtx = mock(BaseEventContext.class);
 
-    return new ResultToMessageIterator(list.iterator(), cursorProviderFactory, eventCtx);
+    return new ResultToMessageIterator(list.iterator(), cursorProviderFactory, eventCtx, eventCtx.getOriginatingLocation());
```

Comment:
i know this is a test, but this may be confusing if looking for examples of this. Can you pass a mock or new location for this?